seedbuilder/areas.ori

```diff
@@ -1793,16 +1793,24 @@ home: MoonGrottoAboveTeleporter
 	pickup: LeftGrottoTeleporterExp
 		casual-core WallJump DoubleJump
 		casual-core Climb DoubleJump ChargeJump
+		standard-abilities DoubleJump Dash Ability=3
 		expert-core Grenade Bash
 		expert-core Climb ChargeJump
+		expert-core DoubleJump Dash
 		expert-dboost ChargeJump Health=4
 		expert-dboost WallJump Glide Health=4
 		expert-dboost Climb Glide Health=4
 		expert-dboost DoubleJump Glide Health=4
```

Comment:
Under the double jump wall climb rules we eventually landed on for expert vs master, we can actually change this path to `expert-dboost DoubleJump Health=4`. Just double jump faceplant the spikes and then press jump again.